core/templates/pages/exploration-player-page/services/content-translation-manager.service.spec.ts

```diff
@@ -322,6 +324,14 @@ describe('Content translation manager service', () => {
       writtenTranslations.addWrittenTranslation(
         'ca_1', 'fr', 'unicode', 'fr 2');
 
+      // This throws "Type '"DummyInteraction"' is not assignable to type
+      // '"ImageClickInput" | "NumberWithUnits" | "NumericExpressionInput" |
+      // "NumericInput" | "DragAndDropSortInput" | "ItemSelectionInput" |
+      // "Continue" | "GraphInput" | "EndExploration" | ... 11 more ... |
+      // "RatioExpressionInput"'." We need to suppress this error because
+      // we need to change the value of 'interaction.id' for testing
+      // purposes.
+      // @ts-expect-error
       interaction.id = 'DummyInteraction';
       interaction.customizationArgs = {
         dummyCustArg: {value: [{
```

Comment:
@shavavo, Can you please confirm  that the `unicode_str` on line 358 and 365 should be "first"? (line number are from the committed file)

Reply:
@17sushmita It looks like this service doesn't work for some interaction (Which doesn't have any translatable contents) can you please add a proper custArg and interaction id so that it uses the translated text?

[ref](https://github.com/oppia/oppia/blob/4df7eadfd73124d58e5d2eb5223160061ab32fe4/core/templates/pages/exploration-player-page/services/content-translation-manager.service.ts#L167)